HerPublicWebsite.BusinessLogic/Services/RegularJobs/ReferralFollowUpService.cs

```diff
@@ -0,0 +1,33 @@
+﻿using HerPublicWebsite.BusinessLogic.Models;
+using HerPublicWebsite.BusinessLogic.Services.WorkingDayHelper;
+
+namespace HerPublicWebsite.BusinessLogic.Services.ReferralFollowUp;
```

Comment:
This namespace looks wrong (I think Rider should warn you about this) - same applies to the other files in this directory.